From 66d389a1bc2b0759d525be050a12df855c6b971a Mon Sep 17 00:00:00 2001
From: Haoran Wang <b50027@freescale.com>
Date: Tue, 20 Oct 2015 09:57:14 +0800
Subject: [PATCH 15/22] MA-6450-1 Use right private lib macro.

Use correct private lib macro for bcm4339.

Signed-off-by: Haoran Wang <b50027@freescale.com>

Conflicts:
	wpa_supplicant/Android.mk
---
 wpa_supplicant/Android.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/wpa_supplicant/Android.mk b/wpa_supplicant/Android.mk
index 476c4e0..660676d 100644
--- a/wpa_supplicant/Android.mk
+++ b/wpa_supplicant/Android.mk
@@ -1551,6 +1551,9 @@ ifdef CONFIG_DRIVER_CUSTOM
 LOCAL_STATIC_LIBRARIES := libCustomWifi
 endif
 LOCAL_STATIC_LIBRARIES += lib_driver_cmd_bcmdhd
+ifneq ($(BOARD_WPA_SUPPLICANT_PRIVATE_LIB_BCM),)
+LOCAL_STATIC_LIBRARIES += $(BOARD_WPA_SUPPLICANT_PRIVATE_LIB_BCM)
+endif
 LOCAL_SHARED_LIBRARIES := libc libcutils liblog
 ifdef CONFIG_EAP_PROXY
 LOCAL_STATIC_LIBRARIES += $(LIB_STATIC_EAP_PROXY)
-- 
1.8.0

